### PR TITLE
Add an option to ignore some window classes in window lists

### PR DESCRIPTION
**Pull request by milouse**

### CODE REVIEW

File: quickswitch.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
 
 workspace_number_re = re.compile('^(?P<number>\d+)(?P<name>.*)')
 default_dmenu_command = 'dmenu -b -i -l 20'
+ignore_class_list = []
```
Comment (OliverUv) [inline quickswitch.py]: rename to `window_class_ignore_list`

File: quickswitch.py [hunk 2]
```diff
@@ -219,12 +220,21 @@ def create_lookup_table(windows):
...
+        class_ = ''
+        if window.get('window_properties') and window['window_properties'].get('class'):
+            class_ = window['window_properties']['class']
         if id_ is None:
             # this is not an X window, ignore it.
             continue
         if name.startswith("i3bar for output"):
             # this is an i3bar, ignore it.
             continue
+        must_go = False
```
Comment (OliverUv) [inline quickswitch.py]: rename to `ignore_window`

**Comment by OliverUv** [comment]
Nice feature! I made some comments, just some variable renaming. Please also update the README to include the description of your feature. After this, it should be ready to merge.

**Comment by milouse** [comment]
Thanks for your quick reviewing! Are you ok with the `-C` short option? I'm quite unsure about its ease to understand/use.

**Comment by OliverUv** [comment]
None of the short options are easy to understand :smile: so yes, it's alright.

**Comment by milouse** [comment]
Hi! What's going on? Do you have any new regards concerning this PR?

Cheers,

**Comment by milouse** [comment]
See #14

**Pull request closed without merging**